.github/ISSUE_TEMPLATE.md

```diff
@@ -1,23 +1,35 @@
 <!--- Remove sections that do not apply -->
+
+This issue tracker is only for technical issues related to bitcoin-core.
+
+General bitcoin questions and/or support requests and are best directed to the [Bitcoin StackExchange](https://bitcoin.stackexchange.com).
+
 ### Describe the issue
 
-### Is the issue reproducible?
-#### List steps to reproduce below:
+### Can you reliably reproduce the issue?
+#### If so, please list the steps to reproduce below:
 1. 
 2. 
 3. 
 
-### Expected behavior
+### Expected behaviour
 Tell us what should happen
 
-### Actual behavior
+### Actual behaviour
 Tell us what happens instead
 
-### Any information in the debug.log file related to this issue?
-
-### Screenshots (if available)
+### Screenshots. 
+If the issue is related to the GUI, screenshots can be added to this issue via drag & drop.
 
-### What binary version was used (official or self compiled)
+### What version of bitcoin-core are you using?
+List the version number/commit ID, and if it is an official binary or self compiled.
```

Comment:
Hi, yes I find interesting, for me, is completely new territory.

Greeting´s 
![427347_418593321560647_374744987_n](https://cloud.githubusercontent.com/assets/22643741/19175854/c5d43842-8c3a-11e6-9630-cd683fa7dff4.png)


Dirk L. Feiler